### ISSUE TITLE
Clicking on the # in Read Only mode could create link to section.

### ISSUE THREAD

**Issue by jordwalke** [Sketch-sh/sketch-sh#139]
In many technical docs, each header has a pound sign `#` next to to a title upon hover. When you click it, it changes the URL to be a link to that header section. How convenient that Sketch already has `#` right next to the headers (although originally for a different purpose)! In Read Only mode, those could form links to headers for sharing a particular section of a Sketch. When you navigate to that URL, it loads the sketch as it normally would but scrolls to that section.
Thoughts?

**Comment by thangngoc89** [comment]
yes. I completely agree with this. There gonna be a complete redesign of read/view/edit mode after fork feature is in. The current design has many limitations that's hard to implement such features.
